src/drivers/magnetometer/lis3mdl/lis3mdl_main.cpp

```diff
@@ -69,55 +70,52 @@ lis3mdl::calibrate(LIS3MDL_BUS bus_id)
 }
 
 int
-lis3mdl::info(LIS3MDL_BUS bus_id)
+lis3mdl::info(struct lis3mdl_bus_option &bus)
 {
-	struct lis3mdl_bus_option &bus = find_bus(bus_id);
-
-	PX4_WARN("running on bus: %u (%s)\n", (unsigned)bus.bus_id, bus.devpath);
+	PX4_INFO("running on bus: %u (%s)", (unsigned)bus.bus_id, bus.devpath);
 	bus.dev->print_info();
-	return 1;
+	return 0;
```

Comment:
Use `PX4_OK` and `PX4_ERROR`. This applies for all return values and checks on functions success/fail. This helps keep the logic clean.